aws-cli/aws-cli.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: aws-cli
+spec:
+  workspaces:
+    - name: source
+  params:
+    - name: SCRIPT
+      description: The AWS script to run
+      type: string
+      default: "aws $@"
+    - name: ARGS
+      description: AWS cli arguments to be passed
+      type: array
+      default: ["help"]
+  steps:
+    - name: awscli
+      image: amazon/aws-cli:latest
+      workingDir: "/tekton/home/.aws"
+      script: "$(params.SCRIPT)"
+      args:
+        - "$(params.ARGS)"
+      volumeMounts:
+        - name: aws-secrets
+          mountPath: "/tekton/home/.aws"
+  volumes:
+    - name: aws-secrets
+      secret:
+        secretName: aws-secrets
```

Comment:
is it possible to use workspaces here instead of volumes? https://github.com/tektoncd/pipeline/blob/master/docs/workspaces.md we're trying to avoid using volumes b/c they give folks less control at runtime

Reply:
@bobcatfish we thought of using the volumes so as that users can mount their aws credentials at the right place. There is also one option of mounting the secrets for AWS is via the environment. Could you please suggest which one would be better.
Ref:- https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html

Reply:
Mounting secrets via environment just like https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#using-a-secret-as-an-environment-source

Reply:
We should try to use `workspace` as much as possible in place of `volumes`. You know you want the secrets in a volume, you don't need to force the user to name it specifically, he will be able to bind it at runtime.

Reply:
(as the docs describe in @bobcatfish link)

Reply:
Okay, will update the task to use workspace. :slightly_smiling_face: